app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+    def __repr__(self) -> str:
+        return f"Person(name=\"{self.name}\", age={self.age})"
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person_dict in people:
+        name = person_dict["name"]
```

Comment:
Consider using `person_dict.get('age')` instead of `person_dict['age']` to avoid potential `KeyError` exceptions and make the code cleaner and more concise as per the checklist recommendation .